app/models/enterprise.rb

```diff
@@ -15,6 +15,9 @@ class Enterprise < ApplicationRecord
     medium: { resize_to_fill: [720, 156] },
     large: { resize_to_fill: [1200, 260] },
   }.freeze
+  WHITE_LABEL_LOGO_SIZES = {
+    default: { gravity: "Center", resize: "217x44^", crop: '217x44+0+0' }
```

Comment:
Rails 7 needs these options in a different format.

* https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#active-storage-default-variant-processor-changed-to-vips

Well, I think that we are not using vips yet and therefore it's working now but it will break when we change.

Reply:
Changed: 2b0d3a6134